### DISCUSSION TITLE
How does one obtain roman numerals for chapter numbers in a quarto book?

### DISCUSSION THREAD

**Discussion by n8craig** [quarto-dev/quarto-cli#8775] [Q&A]
### Description

Starting with a generic quarto book, how does one obtain roman numerals for chapter numbers?

Apologies in advance if I'm missing something in documentation, or an example in the wild--but I'm unclear how to control chapter numbering style. From the sample gallery, the volume [Hands On Programming in R](https://jjallaire.github.io/hopr/) has letters for appendices, but I don't see anything special in the book's [`_quarto.yml`](https://github.com/jjallaire/hopr/blob/master/_quarto.yml).

```
---
crossref:
  chapters: true
  labels: roman
---
```

Putting the above in `_quarto.yml` makes `H1` a chapter if I add another H1 item on any page. However, that new chapter is not included in sidebar. Additionally, chapter labels are still `alpha` but if I add a figure then figure labels are `roman`.

In the documentation on [cross-references](https://quarto.org/docs/authoring/cross-reference-options.html), I see discussion of how to change figure, table, and other kinds of labels but not chapter. Likewise, documentation on [custom reference types ](https://quarto.org/docs/authoring/cross-references-custom.html)provides examples of videos, but I'm not sure how this might be applied to customize chapter numbering.

How would one set the chapter labels to anything other than the default `alpha`? Thanks in advance.

**Comment by mcanouil** [comment]
What defines chapters numerals is being a "chapter" or an "appendix":
- https://github.com/jjallaire/hopr/blob/ef0c5b32ae2f60d17fa1a1e21a8f6ea57da6a548/_quarto.yml#L33

I don't think you can control this, at least for `html` based formats.
